socket/control.go

```diff
@@ -0,0 +1,84 @@
+package socket
```

Comment:
I moved `socket/` back to `control/` but now receive the following failing lint rule...

```
control.ControlSocket [is used] by other packages, and that stutters; consider calling this Socket
```

I feel like `control.Server` gives better context within the application since we're serving HTTP.  I know `Telemetry` does also.

Thoughts?